src/ext/property-panel/__test__/settings.spec.js

```diff
@@ -61,6 +61,36 @@ describe('settings', () => {
       });
     });
 
+    describe('headerFontSize/fontSize/headerFontColor/fontColor.defaultValue', () => {
+      const { headerFontSize, headerFontColor } = headerSection.items.headerFontItem.items.headerFontWrapper.items;
+      const { contentFontSize, contentFontColor } = contentSection.items.contentFontItem.items.contentFontWrapper.items;
+      let args;
+
+      beforeEach(() => {
+        args = {
+          theme: {
+            current: () => {
```

Comment:
Is the case where you get the font-size/color from the straight table theme tested some where else? I'm not seeing it here

Reply:
no, I haven't tested that case, I'll add it now before merging 👍